blog/2024-12-04-area-squared-units.md

```diff
@@ -0,0 +1,25 @@
+---
+author: mikey0000
+authorURL: https://github.com/mikey0000
+title: "Area squared unit enumerators"
+---
+
+As of Home Assistant Core 2024.12, the following unit constant is deprecated and replaced by a corresponding enum:
+
+- `UnitOfArea` enumerator replaces `AREA_SQUARE_METERS` constant
+
+Area units can also be converted between us customary and metric units using `AreaConverter`
+
+# Example
+
+```python
+AreaConverter.convert(10, UnitOfArea.SQUARE_KILOMETERS, UnitOfArea.SQUARE_MILES)
+```
```

Comment:
Is this a real-life example?
I would expect all the unit conversion to happen automatically via the device class - in most cases there should be no need to use the converter.

I'm not sure an example is necessary, but if you do one the focus should be on the `_attr_device_class`

Reply:
I agree. I think the example should be removed or a dev might think they need to handle it (which they should normally not)

Reply:
Agreed, good point will remove